core/src/components/input/input.ios.scss

```diff
@@ -4,6 +4,7 @@
 :host {
   --border-width: #{$hairlines-width};
   --border-color: #{$item-ios-border-color};
+  --highlight-height: 0px;
```

Comment:
This doesn't need to be defined since the highlight is defined in `md` but I kept it here in case we ever move the highlight to the base file.